src/components/migrationToV3.stories.mdx

```diff
@@ -24,6 +24,9 @@ npm install --save amazon-chime-sdk-component-library-react@beta amazon-chime-sd
 - Remove legacy metrics `videoDownstreamGoogFrameHeight`, `videoDownstreamGoogFrameWidth`, `videoUpstreamGoogFrameHeight` and `videoUpstreamGoogFrameWidth` from the `videoStreamMetrics` returned by the `useMediaStreamMetrics` hook to adopt to Amazon Chime SDK for JavaScript V3 changes ([aws/amazon-chime-sdk-js#2086](https://github.com/aws/amazon-chime-sdk-js/pull/2086)).
 - Update the `compilerOptions.target` in `tsconfig.json` from `es5` to `ES2015 (ES6)`.
 - Rename the `global` property of `DefaultTheme` Interface to `globalStyle` to avoid conflict with reserved keyword `global`.
+- Removed `MeetingSessionConfiguration` props from `MeetingProvider`.
+- Added `MeetingSessionConfiguration` as a required prop for the `MeetingManager`'s `join` method.
+- Removed `logLevel`, `postLogConfig`, `logger`, `enableWebAudio`, and `activeSpeakerPolicy` from `MeetingProvider` props and these props are now part of the `options` of `MeetingJoinData` for `join` method.
```

Comment:
Need to use present tense to keep them consistent. We use present tense in CHANGELOG too.